ui/pages/confirmations/components/signature-request-siwe/signature-request-siwe.js

```diff
@@ -68,7 +68,7 @@ export default function SignatureRequestSIWE({ txData }) {
 
   const isLedgerWallet = useSelector((state) => isAddressLedger(state, from));
 
-  const fromAccount = getAccountByAddress(allAccounts, from);
+  const fromAccount = allAccounts.find((account) => account.address === from);
```

Comment:
can we use `[getInternalAccountByAddress](https://github.com/MetaMask/metamask-extension/blob/fix/update-getMetaMaskIdentities-to-getInternalAccounts/ui/selectors/selectors.js#L329)` here?